test/integration/test_ecr.py

```diff
@@ -0,0 +1,44 @@
+import os
+
+import httpretty
+
+from test import HokusaiIntegrationTestCase
+from hokusai.ecr import ECR
+
+httpretty.enable()
+httpretty.HTTPretty.allow_net_connect = False
+
+class TestECR(HokusaiIntegrationTestCase):
+  def setUp(self):
+    self.ecr = ECR()
+
+  @httpretty.activate
+  def test_registry(self):
+    httpretty.register_uri(httpretty.POST, "https://ecr.us-east-1.amazonaws.com/",
+                            body=open(os.path.join(os.getcwd(), 'test', 'fixtures', 'ecr-repositories-response.json')).read(),
+                            content_type="application/x-amz-json-1.1")
+    repositories = [str(repo['repositoryName']) for repo in self.ecr.registry()]
+    self.assertTrue('foo' in repositories)
+    self.assertTrue('bar' in repositories)
+    self.assertTrue('baz' in repositories)
+
+  @httpretty.activate
+  def test_project_repository_exists(self):
+    httpretty.register_uri(httpretty.POST, "https://ecr.us-east-1.amazonaws.com/",
+                            body=open(os.path.join(os.getcwd(), 'test', 'fixtures', 'ecr-repositories-response.json')).read(),
+                            content_type="application/x-amz-json-1.1")
+    self.assertTrue(self.ecr.project_repository_exists())
+
+  @httpretty.activate
+  def test_create_project_repository(self):
+    httpretty.register_uri(httpretty.POST, "https://ecr.us-east-1.amazonaws.com/",
+                            body=open(os.path.join(os.getcwd(), 'test', 'fixtures', 'ecr-repositories-response.json')).read(),
```

Comment:
shouldn't this be the `create-respositories-response`? I would assert for existence of the `createdAt` property in the response, it's too generic currently.